flexiblepricing/views/v0/__init__.py

```diff
@@ -63,6 +62,30 @@ def get_queryset(self):
         if status_search is not None:
             queryset = queryset.filter(status=status_search)
 
+        courseware_search = self.request.query_params.get("courseware")
+        if courseware_search is not None:
+            courseware_content_type, courseware_object_id = courseware_search.split(":")
+            course_type = models.ContentType.objects.get(
```

Comment:
`models` is basically flexiblepricing.models and we are using an import of flexiblepricing/models.py. We should import ContentType using `from django.contrib.contenttypes.models import ContentType`.